.github/workflows/macOS.yml

```diff
@@ -41,44 +58,40 @@ jobs:
           qmake PYTHON_CONFIG=/Library/Frameworks/Python.framework/Versions/${PYTHON_VER_SHORT}/bin/python${PYTHON_VER_SHORT}m-config
           make
           sudo make install
-          cd ..
 
       - name: Build yubioath-desktop
         run: |
           qmake
           make
           macdeployqt yubioath-desktop.app/ -qmldir=qml/ -appstore-compliant
 
-      - name: Copy over dylibs from homebrew
+      - name: Copy over dynamic libraries
         run: |
           sudo find /usr/local/Cellar/json-c/ -name '*.dylib' -exec cp '{}' yubioath-desktop.app/Contents/Frameworks/ ';'
           sudo find /usr/local/Cellar/ykpers/ -name '*.dylib' -exec cp '{}' yubioath-desktop.app/Contents/Frameworks/ ';'
           sudo find /usr/local/Cellar/libyubikey/ -name '*.dylib' -exec cp '{}' yubioath-desktop.app/Contents/Frameworks/ ';'
           sudo find /usr/local/Cellar/libusb/ -name '*.dylib' -exec cp '{}' yubioath-desktop.app/Contents/Frameworks/ ';'
-          sudo find /usr/local/Cellar/openssl@1.1/ -name '*.dylib' -exec cp '{}' yubioath-desktop.app/Contents/Frameworks/ ';'
+          sudo find /opt/openssl/ -name '*.dylib' -exec cp '{}' yubioath-desktop.app/Contents/Frameworks/ ';'
 
       - name: Copy over Python.framework to app bundle
         run: |
           cp -a /Library/Frameworks/Python.framework yubioath-desktop.app/Contents/Frameworks/
           sudo find yubioath-desktop.app/Contents/Frameworks/Python.framework -name '*.pyc' -delete
           sudo find yubioath-desktop.app/Contents/Frameworks/Python.framework -name '__pycache__' -delete
 
-      - name: Move python dependencies to site-packages (required by codesign)
```

Comment:
Any reason why this was removed? Is it because we do not have to codesign when uploading to macOS app store?

Reply:
We do it earlier in the build now, installing them in site-packages in the custom Python we compiled.